wookiespeak.txt

```diff
@@ -0,0 +1,8 @@
+AAAAAAAAAAAAAA
+AAAAAAAAAARRGHHH
+AAAAAAARRGHHH
+
+
+WOOOOOOLLOLOLOOOOO
```

Comment:
Sounds a bit more like a priest from age of empires rather than a wookie but I'll allow it this time

Reply:
What a game!